src/main/java/com/singlestore/jdbc/client/column/StringColumn.java

```diff
@@ -75,8 +75,8 @@ public StringColumn useAliasAsName() {
   }
```

Comment:
What do you think about having a separate Codec for BINARY and VARBINARY?

Reply:
Do you mean a separate column for these types? These types are all in the same place because they have the same encoding/decoding functionality the difference is only metadata.